gradio/__init__.py

```diff
@@ -82,19 +82,18 @@
 from gradio.routes import Request, mount_gradio_app
 from gradio.templates import (
     Files,
-    ImageMask,
-    ImagePaint,
+    # ImageMask,
+    # ImagePaint,
     List,
     Matrix,
     Mic,
     Microphone,
     Numpy,
-    Paint,
-    Pil,
+    # Paint,
+    # Pil,
     PlayableVideo,
-    Sketchpad,
+    # Sketchpad,
```

Comment:
So we don't actually have the ImageEditor in this PR, so these templates don't make much sense. I will bring them back but they might not make a huge amount of sense in v4. We can see.

I don't think the `Webcam` templates really makes any sense now so I did remove that one. I also think `Pil` is a little excessive.